airbyte-integrations/connector-templates/source-configuration-based/source_{{snakeCase name}}/{{snakeCase name}}.yaml.hbs

```diff
@@ -1,42 +1,44 @@
-schema_loader:
-  type: JsonSchema
-  file_path: "./source_{{snakeCase name}}/schemas/\{{ options['name'] }}.json"
-selector:
-  type: RecordSelector
-  extractor:
-    type: JelloExtractor
-    transform: "_"
-requester:
-  type: HttpRequester
-  name: "\{{ options['name'] }}"
-  http_method: "GET"
-  authenticator:
-    type: BearerAuthenticator
-    api_token: "\{{ config['api_key'] }}"
-retriever:
-  type: SimpleRetriever
-  $options:
-    url_base: TODO "your_api_base_url"
-  name: "\{{ options['name'] }}"
-  primary_key: "\{{ options['primary_key'] }}"
-  record_selector:
-    $ref: "*ref(selector)"
-  paginator:
-    type: NoPagination
-customers_stream:
-  type: DeclarativeStream
-  $options:
-    name: "customers"
-  primary_key: "id"
+definitions:
   schema_loader:
-    $ref: "*ref(schema_loader)"
+    type: JsonSchema
+    file_path: "./source_{{snakeCase name}}/schemas/\{{ options['name'] }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      type: JelloExtractor
+      transform: "_"
+  requester:
+    type: HttpRequester
+    name: "\{{ options['name'] }}"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "\{{ config['api_key'] }}"
   retriever:
-    $ref: "*ref(retriever)"
-    requester:
-      $ref: "*ref(requester)"
-      path: TODO "your_endpoint_path"
+    type: SimpleRetriever
+    $options:
+      url_base: TODO "your_api_base_url"
+    name: "\{{ options['name'] }}"
+    primary_key: "\{{ options['primary_key'] }}"
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+
 streams:
-  - "*ref(customers_stream)"
+  - type: DeclarativeStream
+    $options:
+      name: "customers"
+    primary_key: "id"
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        path: TODO "your_endpoint_path"
 check:
   type: CheckStream
   stream_names: ["customers"]
+
+version: "0.0.0-alpha"
```

Comment:
@sherifnada do you have an opinion on what this should be set to?

Reply:
0.1.0 and I would not use alpha. Semver already covers it in the 0.x version